src/actions/content.js

```diff
@@ -97,7 +97,143 @@ function* loadContent(action) {
     yield put(hideLoading());
   }
 }
+export const ACTION_EXECUTE = 'ACTION_EXECUTE';
+export const actionExecute = (action, nids) => {
+  return {
+    type: ACTION_EXECUTE,
+    payload: {
+      action,
+      nids,
+    },
+  };
+};
+
+export const KNOWN_ACTIONS = [
```

Comment:
Would it be more accurate to call this `SUPPORTED_ACTIONS`?

Reply:
Good idea 👍